helpers/deploymentConfig.ts

```diff
@@ -3000,3 +3003,20 @@ export async function getTokenAddress(tokenConfig: TokenConfig, deployments: Dep
     return tokenConfig.tokenAddress;
   }
 }
+
+export const getVWNativeToken = (networkName: string) => {
```

Comment:
I would move this function to the deployment script 018, the only place where we are using it now. In the future, if we need this in any other place, we could move it to one shared location

Reply:
Done https://github.com/VenusProtocol/isolated-pools/pull/361/commits/b508cb1ea7a7fd40bba305ac47aca94f903aae9f